frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -0,0 +1,471 @@
+    }
+    if (isApprovalNeeded) {
+      return (
+        <InstallNeedsApprovalMessage
+          namespace={targetNamespace}
+          obj={subscription}
+          approve={approve}
+        />
+      );
+    }
+    if (isStatusSucceeded) {
+      return (
+        <InstallSucceededMessage
+          namespace={targetNamespace}
+          csvName={pkgNameWithVersion}
+          obj={installObj}
+        />
+      );
+    }
+    return <InstallingMessage namespace={targetNamespace} obj={installObj} />;
+  };
+
+  const getCSVLogo = () => {
+    const pkgManifest = resources.packageManifest.data[0];
+    if (!pkgManifest) {
+      return null;
+    }
+    const channels = pkgManifest.status?.channels || [];
+    const channel = channels.find((ch) => ch.currentCSV === pkgNameWithVersion) || channels[0];
+    const displayName = channel?.currentCSVDesc?.displayName || '';
+    const logoVersion = channel?.currentCSVDesc?.version || '';
+
+    return (
+      <ClusterServiceVersionLogo
+        displayName={displayName}
+        icon={iconFor(pkgManifest)}
+        provider={pkgManifest.status?.provider?.name || ''}
+        version={logoVersion}
+      />
+    );
+  };
+
+  return (
+    <>
+      <div className="co-operator-install-page__main">
+        <Helmet>
+          <title>View Install</title>
+        </Helmet>
+        <Bullseye>
+          <div id="operator-install-page">
+            {loading && (
+              <div>
+                Installing... <Spinner size="lg" />
+              </div>
+            )}
+            {!loading && isStatusFailed && (
+              <Alert variant="danger" isInline title="Installation Failed">
+                {status}: {(installObj as ClusterServiceVersionKind)?.status?.message || ''}
+              </Alert>
+            )}
+            {!loading && (
+              <Card>
+                <CardBody>
+                  <div className="co-operator-install-page__pkg-indicator">
+                    <div>{getCSVLogo()}</div>
+                    <div>{getIndicator()}</div>
+                  </div>
+                </CardBody>
+              </Card>
+            )}
+            {!loading && (
+              <Card>
+                <CardBody>{getInstallMessage()}</CardBody>
+              </Card>
+            )}
+          </div>
+        </Bullseye>
+      </div>
+      {!loading && isApprovalNeeded && (
+        <InstallPlanPreview obj={installObj as InstallPlanKind} hideApprovalBlock />
+      )}
+    </>
+  );
+};
+
+export const OperatorInstallPage: React.FC<OperatorInstallPageProps> = (props) => {
+  const { pkgNameWithVersion, targetNamespace } = props;
+  const namespace = targetNamespace;
+  const pkg = new URLSearchParams(window.location.search).get('pkg');
+  const installPageResources = [
+    {
+      kind: referenceForModel(ClusterServiceVersionModel),
+      namespaced: true,
+      fieldSelector: `metadata.name=${pkgNameWithVersion}`,
```

Comment:
Instead of using `fieldSelector`, why not `isList: false, name: pkgNameWithVersion` (here and below).

Reply:
I did not realize you could just use `name`.  As for `isList`, another left over from when I was grabbing resources from global and namespace.